src/ansibleCompletionItemProvider.ts

```diff
@@ -22,6 +22,11 @@ export class AnsibleCompletionItemProvider implements vscode.CompletionItemProvi
         let prefix = range ? document.getText(range) : '';
         let lineText = document.lineAt(position.line).text;
 
+        var index = lineText.indexOf(':');
```

Comment:
i think this might require some comment how it works exactly. a kind of difficult without debugging...